typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -288,38 +292,6 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     );
   }
 
-  updateProxyAdminOwnershipTxs(
```

Comment:
nit: you can always keep this function and just call the static proxyAdminOwnershipUpdateTxs() to keep the same function signature of only having actualConfig and expectedConfig